Source/ACE.Database/WorldDatabase.cs

```diff
@@ -39,7 +39,10 @@ private enum WorldPreparedStatement
             GetAceObjectPropertiesPosition = 24,
             GetAceObjectPropertiesSpell = 25,
             GetAceObjectGeneratorLinks = 26,
-            GetMaxId = 27
+            GetMaxId = 27,
+            GetAceObjectPropertiesAttributes,
```

Comment:
Og's PR removes them all, I only added new ones from Shard and did not include numbers